docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -35,85 +35,92 @@ You can set up xCluster replication as follows:
 
 1. Open the YugabyteDB Anywhere UI and navigate to **Universes**.
 
-2. Select the universe you want to replicate and navigate to **Replication**.
+1. Select the universe you want to replicate and navigate to **Replication**.
 
-3. Click **Configure Replication** to open the dialog shown in the following illustration:<br><br>
+1. Click **Configure Replication** to open the dialog shown in the following illustration:<br>
 
-   ![Configure Replication](/images/yp/asynch-replication-2.png)<br><br>
+   ![Configure Replication](/images/yp/asynch-replication-21.png)<br>
 
-4. Provide the name for your replication.
+1. Provide the name for your replication.
 
-5. Select the target universe.
+1. Select the target universe.
 
-6. Click **Next: Select Tables**.
+1. Click **Next: Select Tables**.
 
-7. From a list of common tables between source and target universes, select the tables you want to include in the replication and then click **Create Replication**, as per the following illustration:<br><br>
+1. From a list of common tables between source and target universes, select the tables you want to include in the replication and then click **Validate Table Selection**, as per the following illustration:<br><br>
 
-   ![Create Replication](/images/yp/asynch-replication-3.png)
+   ![Create Replication](/images/yp/asynch-replication-31.png)<br><br>
 
-## Configure replication
+   The replication for the index tables is set up if the main table is selected for replication setup.
 
-You can configure an existing replication as follows:
+1. Optionally, configure alerts on lag, as follows:
 
-1. Open the YugabyteDB Anywhere UI and navigate to **Universes**.
-
-2. Select the universe whose existing replication you want to modify and then navigate to **Replication**, as per the following illustration:<br><br>
+   - Click **Max acceptable lag time**.
+   - Use the **Define Max Acceptable Lag Time** dialog to enable or disable alert issued when the replication lag exceeds the specified threshold, as per the following illustration:<br><br>
+   ![Create Replication](/images/yp/asynch-replication-91.png)
 
-   ![Replication](/images/yp/asynch-replication-1.png)<br><br>
+Bootstrapping is automatically enabled for any new and existing replication. When the source universe has data, its write-ahead logs are garbage-collected. If bootstrapping is not required for tables that are to be added, these tables are added to the replication group. However, if there is a table that requires bootstrapping, all other tables in the same database are removed from the replication group and the replication is set up using the bootstrap flow for all the tables, including the new ones requested to be added. Note that a replication may contain tables from different databases which are not affected by the remove operation.
```

Comment:
Remove this paragraph. Will add the explanation at the View, manage, and monitor replication section for adding/removing a table to/from an xCluster config.

Reply:
Removed